pkg/relayer/proxy/proxy_test.go

```diff
@@ -75,12 +80,12 @@ func init() {
 
 	proxiedServices = map[string]*config.RelayMinerProxyConfig{
 		"server1": {
-			ProxyName: "server1",
+			ProxyName: defaultServer,
```

Comment:
You are right, it's confusing.

The key and `ProxyName` have to be the same, as the config parser maps the proxy servers using their `ProxyName` as keys.

I was uncomfortable having a variable as a key but afterthought, I believe it will add more clarity. Let's having as
```
defaultProxyServer: {
  ProxyName: defaultProxyServer,
  ...
}
```